scripts/exaqm_post_stat_o3.sh

```diff
@@ -145,15 +145,29 @@ if [ "${cyc}" = "06" ] || [ "${cyc}" = "12" ]; then
 fi
 
 grid227="lambert:265.0000:25.0000:25.0000 226.5410:1473:5079.000 12.1900:1025:5079.000"
-grid196="mercator:20.0000 198.4750:321:2500.000:206.1310 18.0730:255:2500.000:23.0880"
+grid196="mercator:20.0000 198.4750:321:2500.000:206.1310 18.0730:225:2500.000:23.0880"
 grid198="nps:210.0000:60.0000 181.4290:825:5953.000 40.5300:553:5953.000"
 
 for grid in 227 196 198;do
   gg="grid${grid}"
-  wgrib2 ${NET}.${cycle}.ave_1hr_o3.${id_domain}.grib2 -set_grib_type c3b -new_grid_winds earth -new_grid ${!gg} ${NET}.${cycle}.ave_1hr_o3.${grid}.grib2
+  wgrib2 ${NET}.${cycle}.ave_1hr_o3.${id_domain}.grib2 -set_grib_type c3b -new_grid_winds grid -new_grid ${!gg} ${NET}.${cycle}.tmp.ave_1hr_o3.${grid}.grib2
+
+  # fix res flags
+  if [ "$grid" == "198" ] || [ "$grid" == "227" ]; then 
+     wgrib2 -set_flag_table_3.3 8 "${NET}.${cycle}.tmp.ave_1hr_o3.${grid}.grib2" -grib "${NET}.${cycle}.ave_1hr_o3.${grid}.grib2"
+  else
+     cp "${NET}.${cycle}.tmp.ave_1hr_o3.${grid}.grib2" "${NET}.${cycle}.ave_1hr_o3.${grid}.grib2"
```

Comment:
cp/cpreq?

Reply:
It is the same as the operational job, exaqm_post_stat_o3.sh at /lfs/h1/ops/prod
/packages/aqm.v7.0.7/scripts/